.obs/chartfile/defaults/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: Apache-2.0
+#!BuildTag: rancher/elemental-operator-defaults-chart:latest
+#!BuildTag: rancher/elemental-operator-defaults-chart:%VERSION%
+#!BuildTag: rancher/elemental-operator-defaults-chart:%VERSION%-%RELEASE%
+apiVersion: v2
+name: elemental-operator-defaults
+description: A Helm chart for deploying Rancher Elemental Operator CRDs
+icon: https://raw.githubusercontent.com/rancher/elemental/main/logo/icon-elemental.svg
+version: "%VERSION%"
+appVersion: "%VERSION%"
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Elemental Operator default resources
+  catalog.cattle.io/experimental: "true"
+  catalog.cattle.io/namespace: cattle-elemental-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/provides-gvr: elemental.cattle.io/v1beta1
```

Comment:
Wondering if we should drop this annotation, as the Elemental GVR is provided by the other chart 🤔 
I was not able anyway to find a doc with Rancher annotations for charts explained.
So, not sure of the implications how having or removing it 🤷🏼 
```suggestion

```